src/test/groovy/net/fabricmc/loom/test/util/ProjectTestTrait.groovy

```diff
@@ -59,7 +60,7 @@ trait ProjectTestTrait {
 			}
 
 			tempFile.parentFile.mkdirs()
-			tempFile << file.text
+			Files.copy(file, tempFile)
```

Comment:
The groovy way of doing this would be `tempFile.bytes = file.bytes`

Reply:
Sure, I'll change it. (Though if the files were a bit larger, copying would probably require less memory since it streams the content gradually.)

Reply:
Thats true, its fine either way. Just not a massive fan using libs that I 1/2 plan on removing at somepoint.